src/example.js

```diff
@@ -21,6 +21,9 @@ export default {
       content: '"icon"'
     }
   },
+  someClassName: {
```

Comment:
we call it rule name, because class name is generated, but please use something more meaningful

Reply:
@kof meaningful name added